core-api/src/main/java/com/optimizely/ab/config/PollingProjectConfigManager.java

```diff
@@ -203,6 +208,10 @@ public synchronized void close() {
         started = false;
     }
 
+    protected void setSdkKey(String sdkKey) {
+        this.sdkKey = sdkKey;
+    }
+
```

Comment:
What about moving `getSdkKey` from HttpProjectConfigManager to here? It make look better with get/set combo here and HttpProjectConfigManager inherit both.